react_native/src/screens/root_screens/RootStackScreenComponent.js

```diff
@@ -16,63 +16,125 @@ const Stack = createNativeStackNavigator();
 const Tab = createBottomTabNavigator();
 
 function RootStackScreenComponent() {
-    const loggedIn = true;
-    return (
-        <ImageBackground source={backgroundLight} resizeMode="cover" style={{ width: "100%", flex: 1, justifyContent: "center" }}>
-            {!loggedIn ?
-                <Stack.Navigator screenOptions={{headerShown: false}}>
-                    <Stack.Screen name={"SplashScreen"} component={SplashScreenComponent}/>
-                    <Stack.Screen name={"SignIn"} component={SignInScreenComponent}  options={{headerBackTitle: "Back"}}/>
-                    <Stack.Screen name={"SignUp"} component={SignUpScreenComponent}  options={{headerBackTitle: "Back"}}/>
-                </Stack.Navigator> :
-                <Animatable.View animation={"fadeInUp"} style={{flex: 1}}>
-                    <Tab.Navigator
-                        initialRouteName="Home"
-                        screenOptions={{
-                            backgroundColor: "#ffff",
-                            showLabel: false,
-                            tabBarStyle: {
-                                position: "absolute",
-                                alignItems: "center",
-                                backgroundColor: "#ffffff",
-                                height: 90,
-                            }
-                        }}
-                    >
-                        <Tab.Screen
-                            name={"Home"}
-                            component={HomeScreenComponent}
-                            options={{
-                                headerShown: false,
-                                tabBarLabel: ({focused}) => (<Text style={{fontSize: 10, color: focused ? "#000000" : "#696969"}}>Home</Text>),
-                                tabBarIconStyle: {top: 5},
-                                tabBarIcon: ({focused}) => (<Icon name={'home'} color={focused ? "#000000" : "#696969"} size={26}/>)
-                            }}
-                        />
-                        <Tab.Screen
-                            name={"Wallet"}
-                            component={WalletScreenComponent}
-                            options={{
-                                headerShown: false,
-                                tabBarLabel: ({focused}) => (<Text style={{fontSize: 10, color: focused ? "#000000" : "#696969"}}>Wallet</Text>),
-                                tabBarIconStyle: {top: 5},
-                                tabBarIcon: ({focused}) => (<Icon name={'wallet'} color={focused ? "#000000" : "#696969"} size={26}/>)
-                            }}
-                        />
+  const loggedIn = true;
+  return (
+    <ImageBackground
+      source={backgroundLight}
+      resizeMode="cover"
+      style={{ width: "100%", flex: 1, justifyContent: "center" }}
+    >
+      {loggedIn ? ( // manual toggle for now to test
```

Comment:
Why would we show the root splash navigator if the user is logged in instead of the main home screen navigator?

Reply:
@NataliaLopez17